lib/constants.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const sizes = {
+  HANDSHAKE_SIZE: 36,
+  PARCEL_HEADER_SIZE: 16,
+  CHUNK_HEADER_SIZE: 12,
+};
+
+const structTypes = {
+  STRUCT_PARCEL_HEADER: 0,
```

Comment:
```suggestion
  PARCEL_HEADER: 0,
```
IMHO

Reply:
@mille-nium WDYT?

Reply:
As far as all types and sizes are exported in one namespace, there will be some misconceptions in naming later when requiring these constants, if we remove `STRUCT` and `PARCEL` prefix, as for me.